cfgov/unprocessed/css/cf-enhancements.less

```diff
@@ -1390,6 +1390,33 @@ textarea {
 
 
 /* topdoc
+  name: Invisible
+  family: cf-core
+  patterns:
+    - name: Utility class
+      markup: |
+        <h1>
+            <a href="#">
+                <span class="u-invisible">Not shown</span>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/d152faa87c74af775afa753df5bec8557e7fe1ca